tests/unit/composables/Project.spec.js

```diff
@@ -250,4 +253,18 @@ describe('Test composable: Project', () => {
       ]);
     });
   });
+
+  describe('Test function: newBranchFrom', () => {
+    it('Should not call checkout without checkout demand', async () => {
```

Comment:
`call checkout` => `call checkout function`

Reply:
`without checkout demand` => `when haveToCheckout is false`

Reply:
done